CRITICAL - 2024-03-25 03:41:08 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:41:13 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Views\footer.php(19): company('2')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\footer.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('footer', [], true)
 6 APPPATH\Controllers\Settings.php(87): view('footer')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:41:42 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:41:57 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:42:03 --> Undefined variable $user
in APPPATH\Views\settings\account_settings.php on line 48.
 1 APPPATH\Views\settings\account_settings.php(48): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'APPPATH\\Views\\settings\\account_settings.php', 48)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 5 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:42:14 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:43:06 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:53:35 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:54:27 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:56:34 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:56:58 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Views\footer.php(19): company('2')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\footer.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('footer', [], true)
 6 APPPATH\Controllers\Settings.php(87): view('footer')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:57:21 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Controllers\Settings.php(68): company(38)
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:58:04 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(77): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:58:26 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(77): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:58:34 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(77): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:59:07 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 03:59:54 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Views\footer.php(19): company('2')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\footer.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('footer', [], true)
 6 APPPATH\Controllers\Settings.php(87): view('footer')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:01:28 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:03:54 --> Trying to access array offset on value of type null
in APPPATH\Helpers\apanel_helper.php on line 5010.
 1 APPPATH\Helpers\apanel_helper.php(5010): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Helpers\\apanel_helper.php', 5010)
 2 APPPATH\Helpers\apanel_helper.php(4972): company('2')
 3 APPPATH\Views\settings\account_settings.php(196): now_time('2')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\account_settings.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/account_settings', [], true)
 7 APPPATH\Controllers\Settings.php(86): view('settings/account_settings')
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:05:41 --> mysqli_sql_exception: Column 'id' in where clause is ambiguous in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `main_it...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `main_it...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `main_it...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_it...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(66): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->index()
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-03-25 04:05:41 --> Column 'id' in where clause is ambiguous
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_item_party_table`.*, `item_party_meta_1`.*, `item_party_meta_2`.*
FROM `main_item_party_table`
LEFT JOIN `item_party_meta_1` ON `item_party_meta_1`.`main_id` = `main_item_party_table`.`id`
LEFT JOIN `item_party_meta_2` ON `item_party_meta_2`.`main_id` = `main_item_party_table`.`id`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Settings.php(66): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:06:03 --> mysqli_sql_exception: Unknown table 'aitsun_app_db.item_party_meta_1' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `main_it...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `main_it...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `main_it...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_it...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(66): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->index()
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-03-25 04:06:03 --> Unknown table 'aitsun_app_db.item_party_meta_1'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_item_party_table`.*, `item_party_meta_1`.*, `item_party_meta_2`.*
FROM `main_item_party_table`
WHERE `main_item_party_table`.`id` = :main_item_party_table.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Settings.php(66): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:21:17 --> Undefined variable $Item_party_meta_1
in APPPATH\Controllers\Settings.php on line 257.
 1 APPPATH\Controllers\Settings.php(257): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $Item_party_meta_1', 'APPPATH\\Controllers\\Settings.php', 257)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:21:25 --> Undefined variable $Item_party_meta_1
in APPPATH\Controllers\Settings.php on line 257.
 1 APPPATH\Controllers\Settings.php(257): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $Item_party_meta_1', 'APPPATH\\Controllers\\Settings.php', 257)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:23:17 --> Undefined variable $Item_party_meta_1
in APPPATH\Controllers\Settings.php on line 257.
 1 APPPATH\Controllers\Settings.php(257): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $Item_party_meta_1', 'APPPATH\\Controllers\\Settings.php', 257)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:23:17 --> Undefined variable $Item_party_meta_1
in APPPATH\Controllers\Settings.php on line 257.
 1 APPPATH\Controllers\Settings.php(257): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $Item_party_meta_1', 'APPPATH\\Controllers\\Settings.php', 257)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:23:18 --> Undefined variable $Item_party_meta_1
in APPPATH\Controllers\Settings.php on line 257.
 1 APPPATH\Controllers\Settings.php(257): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $Item_party_meta_1', 'APPPATH\\Controllers\\Settings.php', 257)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:23:18 --> Undefined variable $Item_party_meta_1
in APPPATH\Controllers\Settings.php on line 257.
 1 APPPATH\Controllers\Settings.php(257): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $Item_party_meta_1', 'APPPATH\\Controllers\\Settings.php', 257)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:25:07 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
in SYSTEMPATH\BaseModel.php on line 1706.
 1 SYSTEMPATH\BaseModel.php(910): CodeIgniter\BaseModel->transformDataToArray('1', 'update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], '1')
 3 APPPATH\Controllers\Settings.php(259): CodeIgniter\Model->update('main_id', '1')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:25:09 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
in SYSTEMPATH\BaseModel.php on line 1706.
 1 SYSTEMPATH\BaseModel.php(910): CodeIgniter\BaseModel->transformDataToArray('1', 'update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], '1')
 3 APPPATH\Controllers\Settings.php(259): CodeIgniter\Model->update('main_id', '1')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:25:09 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
in SYSTEMPATH\BaseModel.php on line 1706.
 1 SYSTEMPATH\BaseModel.php(910): CodeIgniter\BaseModel->transformDataToArray('1', 'update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], '1')
 3 APPPATH\Controllers\Settings.php(259): CodeIgniter\Model->update('main_id', '1')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:27:54 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
in SYSTEMPATH\BaseModel.php on line 1706.
 1 SYSTEMPATH\BaseModel.php(910): CodeIgniter\BaseModel->transformDataToArray('1', 'update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], '1')
 3 APPPATH\Controllers\Settings.php(259): CodeIgniter\Model->update('main_id', '1')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:29:09 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
in SYSTEMPATH\BaseModel.php on line 1706.
 1 SYSTEMPATH\BaseModel.php(910): CodeIgniter\BaseModel->transformDataToArray('1', 'update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], '1')
 3 APPPATH\Controllers\Settings.php(259): CodeIgniter\Model->update('main_id', '1')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:40:17 --> Undefined variable $main_id
in APPPATH\Controllers\Settings.php on line 261.
 1 APPPATH\Controllers\Settings.php(261): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $main_id', 'APPPATH\\Controllers\\Settings.php', 261)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:40:39 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 04:40:39 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('1', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:40:46 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 04:40:46 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('1', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:42:36 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 04:42:36 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('1', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:42:38 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 04:42:38 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('1', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:42:38 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 04:42:38 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('1', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:42:39 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 04:42:39 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('1', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 04:44:49 --> Undefined array key "id"
in APPPATH\Controllers\Settings.php on line 261.
 1 APPPATH\Controllers\Settings.php(261): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'APPPATH\\Controllers\\Settings.php', 261)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 04:45:41 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 04:45:41 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:35:42 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:35:42 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:38:55 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:38:55 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:40:32 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update(2, Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:40:32 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update(2, [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 05:40:42 --> The action you requested is not allowed.
in SYSTEMPATH\Security\Security.php on line 306.
 1 SYSTEMPATH\Security\Security.php(306): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
 2 SYSTEMPATH\Filters\CSRF.php(55): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
 3 SYSTEMPATH\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 4 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('settings/save_profile/1', 'before')
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 05:41:24 --> The action you requested is not allowed.
in SYSTEMPATH\Security\Security.php on line 306.
 1 SYSTEMPATH\Security\Security.php(306): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
 2 SYSTEMPATH\Filters\CSRF.php(55): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
 3 SYSTEMPATH\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 4 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('settings/save_profile/1', 'before')
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:41:45 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update(2, Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:41:45 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update(2, [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:44:22 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:44:22 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:45:48 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:45:48 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:46:23 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update(2, Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:46:23 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update(2, [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:47:25 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:47:25 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:47:27 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:47:27 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:47:34 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:47:34 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:47:39 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:47:39 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 05:47:43 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `item_pa...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `item_pa...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `item_pa...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_pa...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(261): CodeIgniter\Model->update('2', Array)
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 05:47:43 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `item_party_meta_1` SET `billing_address` = :billing_address:, `designation` = :designation:
WHERE `item_party_meta_1`.`id` IN :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Settings.php(261): CodeIgniter\Model->update('2', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->save_profile('1')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 06:10:05 --> Class "App\Controllers\UserModel" not found
in APPPATH\Controllers\Company.php on line 28.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Company->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Company))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 06:21:28 --> mysqli_sql_exception: Unknown column 'item_party_meta_2.main_id' in 'on clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `main_it...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `main_it...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `main_it...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_it...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\aitsun-erp\app\Helpers\apanel_helper.php(5073): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(4605): user_data('1', 'activated_finan...')
#8 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->change_academic_year('31')
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-03-25 06:21:28 --> Unknown column 'item_party_meta_2.main_id' in 'on clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_item_party_table`.*, `item_party_meta_1`.*, `item_party_meta_2`.*
FROM `main_item_party_table`
LEFT JOIN `item_party_meta_1` ON `item_party_meta_1`.`main_id` = `main_item_party_table`.`id`
LEFT JOIN `item_party_meta_2` ON `item_party_meta_2`.`main_id` = `main_item_party_table`.`id`
WHERE `main_item_party_table`.`id` = :main_item_party_table.id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 4 APPPATH\Helpers\apanel_helper.php(5073): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Settings.php(4605): user_data('1', 'activated_financial_year')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->change_academic_year('31')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 07:19:37 --> Undefined array key "item_party_meta_1.year_end"
in APPPATH\Helpers\auth_helper.php on line 77.
 1 APPPATH\Helpers\auth_helper.php(77): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "item_party_meta_1.year_end"', 'APPPATH\\Helpers\\auth_helper.php', 77)
 2 APPPATH\Views\settings\financial_years.php(16): company_year_end('39')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\financial_years.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/financial_years', [], true)
 6 APPPATH\Controllers\Settings.php(614): view('settings/financial_years', [...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->financial_years()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-25 07:19:52 --> mysqli_sql_exception: Unknown column 'item_party_meta_1.id' in 'where clause' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `main_it...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `main_it...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `main_it...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_it...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\aitsun-erp\app\Helpers\auth_helper.php(75): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\aitsun-erp\app\Views\settings\financial_years.php(16): company_year_end('39')
#8 C:\xampp\htdocs\aitsun-erp\system\View\View.php(213): include('C:\\xampp\\htdocs...')
#9 C:\xampp\htdocs\aitsun-erp\system\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#10 C:\xampp\htdocs\aitsun-erp\system\Common.php(1184): CodeIgniter\View\View->render('settings/financ...', Array, true)
#11 C:\xampp\htdocs\aitsun-erp\app\Controllers\Settings.php(614): view('settings/financ...', Array)
#12 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Settings->financial_years()
#13 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
#14 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#16 {main}
CRITICAL - 2024-03-25 07:19:52 --> Unknown column 'item_party_meta_1.id' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `main_item_party_table`.*, `item_party_meta_1`.*
FROM `main_item_party_table`
LEFT JOIN `item_party_meta_1` ON `item_party_meta_1`.`main_id` = `main_item_party_table`.`id`
WHERE `item_party_meta_1`.`id` = :item_party_meta_1.id:', [...], false)
 2 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Helpers\auth_helper.php(75): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Views\settings\financial_years.php(16): company_year_end('39')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\settings\\financial_years.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('settings/financial_years', [], true)
 9 APPPATH\Controllers\Settings.php(614): view('settings/financial_years', [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Settings->financial_years()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Settings))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 09:33:49 --> Undefined variable $UserModel
in APPPATH\Controllers\Customers.php on line 30.
 1 APPPATH\Controllers\Customers.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $UserModel', 'APPPATH\\Controllers\\Customers.php', 30)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 09:35:01 --> Class "App\Controllers\AccountCategory" not found
in APPPATH\Controllers\Customers.php on line 113.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->add()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 09:36:46 --> Class "App\Controllers\AccountCategory" not found
in APPPATH\Controllers\Customers.php on line 113.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->add()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 09:37:22 --> Class "App\Controllers\AccountCategory" not found
in APPPATH\Controllers\Customers.php on line 153.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->add_customer_form()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 09:37:31 --> Class "App\Controllers\AccountCategory" not found
in APPPATH\Controllers\Customers.php on line 153.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->add_customer_form()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 09:37:33 --> Class "App\Controllers\AccountCategory" not found
in APPPATH\Controllers\Customers.php on line 153.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->add_customer_form()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 09:58:26 --> Class "App\Controllers\CustomerBalances" not found
in APPPATH\Controllers\Customers.php on line 352.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:01:47 --> Undefined array key "billing_address"
in APPPATH\Views\customers\details.php on line 55.
 1 APPPATH\Views\customers\details.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "billing_address"', 'APPPATH\\Views\\customers\\details.php', 55)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(647): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:05:10 --> Undefined array key "billing_address"
in APPPATH\Views\customers\details.php on line 55.
 1 APPPATH\Views\customers\details.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "billing_address"', 'APPPATH\\Views\\customers\\details.php', 55)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(651): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:05:53 --> Undefined variable $cust
in APPPATH\Views\customers\details.php on line 20.
 1 APPPATH\Views\customers\details.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cust', 'APPPATH\\Views\\customers\\details.php', 20)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(651): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:06:25 --> Undefined array key "billing_address"
in APPPATH\Views\customers\details.php on line 55.
 1 APPPATH\Views\customers\details.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "billing_address"', 'APPPATH\\Views\\customers\\details.php', 55)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(654): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:07:00 --> Undefined array key "billing_address"
in APPPATH\Views\customers\details.php on line 55.
 1 APPPATH\Views\customers\details.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "billing_address"', 'APPPATH\\Views\\customers\\details.php', 55)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(654): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:07:07 --> Undefined array key "billing_address"
in APPPATH\Views\customers\details.php on line 55.
 1 APPPATH\Views\customers\details.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "billing_address"', 'APPPATH\\Views\\customers\\details.php', 55)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(654): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('156')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:07:15 --> Undefined array key "saved_as"
in APPPATH\Views\customers\details.php on line 267.
 1 APPPATH\Views\customers\details.php(267): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "saved_as"', 'APPPATH\\Views\\customers\\details.php', 267)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(654): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('6')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:08:01 --> Undefined array key "billing_address"
in APPPATH\Views\customers\details.php on line 55.
 1 APPPATH\Views\customers\details.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "billing_address"', 'APPPATH\\Views\\customers\\details.php', 55)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\customers\\details.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('customers/details', [], true)
 5 APPPATH\Controllers\Customers.php(654): view('customers/details', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:11:17 --> Undefined variable $allpayments
in APPPATH\Controllers\Customers.php on line 396.
 1 APPPATH\Controllers\Customers.php(396): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $allpayments', 'APPPATH\\Controllers\\Customers.php', 396)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('157')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:16:24 --> Undefined variable $UserModel
in APPPATH\Controllers\Customers.php on line 609.
 1 APPPATH\Controllers\Customers.php(609): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $UserModel', 'APPPATH\\Controllers\\Customers.php', 609)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:29:19 --> Undefined variable $UserModel
in APPPATH\Controllers\Customers.php on line 609.
 1 APPPATH\Controllers\Customers.php(609): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $UserModel', 'APPPATH\\Controllers\\Customers.php', 609)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:46:44 --> Undefined variable $looc
in APPPATH\Controllers\Customers.php on line 629.
 1 APPPATH\Controllers\Customers.php(629): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $looc', 'APPPATH\\Controllers\\Customers.php', 629)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 10:59:14 --> Undefined variable $closing_balance
in APPPATH\Controllers\Customers.php on line 592.
 1 APPPATH\Controllers\Customers.php(592): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $closing_balance', 'APPPATH\\Controllers\\Customers.php', 592)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:05:51 --> Unsupported operand types: string - string
in APPPATH\Controllers\Customers.php on line 575.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:07:06 --> Unsupported operand types: string - string
in APPPATH\Controllers\Customers.php on line 575.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:07:19 --> Unsupported operand types: string - string
in APPPATH\Controllers\Customers.php on line 575.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:07:36 --> syntax error, unexpected token "(double)"
in APPPATH\Controllers\Customers.php on line 432.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Customers.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Customers')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Customers')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Customers', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:08:02 --> Unsupported operand types: string - string
in APPPATH\Controllers\Customers.php on line 575.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:12:25 --> Unsupported operand types: string - string
in APPPATH\Controllers\Customers.php on line 575.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Customers->details('158')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:22:02 --> Class "App\Controllers\Main_item_party_table" not found
in APPPATH\Controllers\Import_and_export.php on line 860.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Import_and_export->export_parties()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Import_and_export))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:50:50 --> Undefined array key "designation"
in APPPATH\Views\user_master\staffs.php on line 377.
 1 APPPATH\Views\user_master\staffs.php(377): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "designation"', 'APPPATH\\Views\\user_master\\staffs.php', 377)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\user_master\\staffs.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('user_master/staffs', [], true)
 5 APPPATH\Controllers\User_master.php(76): view('user_master/staffs')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User_master->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User_master))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-25 11:58:03 --> Attempt to read property "pager" on array
in APPPATH\Controllers\User_master.php on line 71.
 1 APPPATH\Controllers\User_master.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "pager" on array', 'APPPATH\\Controllers\\User_master.php', 71)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User_master->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User_master))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
